test/e2e/specs/blocks/list.test.js

```diff
@@ -155,4 +155,18 @@ describe( 'List', () => {
 
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
+
+	it( 'should split indented list item', async () => {
+		await insertBlock( 'List' );
+		await page.keyboard.type( 'one' );
+		await page.keyboard.press( 'Enter' );
+		await page.mouse.move( 200, 300, { steps: 10 } );
+		await page.mouse.move( 250, 350, { steps: 10 } );
```

Comment:
Why must we move the mouse? Is there a keyboard-only workflow?

Reply:
It didn't work for me otherwise. Shift+Tab shows the toolbar but does not focus it. Any fix is outside the scope of this PR.

Reply:
Can we leave an inline comment if it's undesirable, and/or check for an existing issue?

Would it be fixed by #10906 ?

Reply:
The comment doesn't read as though there's anything bad to be fixed. I'd expect either making it clear in the comment ("todo: fix badness") or create as a separate issue.

Reply:
There are instances of this spread across the tests. I don't really see it as something bad this PR is introducing but an necessity because of how current things works. This is a perfectly valid way to convert a block, by moving them mouse and clicking the button?

I'll update the comment tomorrow.